examples/models/llama/model_args.py

```diff
@@ -49,6 +49,7 @@ class ModelArgs:
     model_architecture: str = (
         "LlamaForCausalLM"  # This setting is currently only supported for the QNN backend
     )
+    attention_multiplier: Optional[float] = None
```

Comment:
Can you add some comments to explain the new arguments for attention_multiplier, logits_scaling and residual_multiplier? I understand we don't have them for the other params, but let's start document them to make it more scalable and readable.